pkg/controller/account/account_controller.go

```diff
@@ -276,18 +276,28 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 	// If the account is BYOC, needs some different set up
 	if currentAcctInstance.Spec.BYOC {
 		reqLogger.Info("BYOC account")
-		//Set awsAccountID
-		awsAccountID = currentAcctInstance.Spec.AwsAccountID
```

Comment:
I assume this is blank? Since we have other logic that looks to see if this is empty should we put the BYOCAWSAccountID here? Though that could be confusing.